tasks/eva_3764/delete_duplicate_rs_ids.py

```diff
@@ -0,0 +1,78 @@
+import sys
+from argparse import ArgumentParser
+from itertools import islice
+
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+from ebi_eva_internal_pyutils.mongo_utils import get_mongo_connection_handle
+
+logger = log_cfg.get_logger(__name__)
+
+
+def chunked_iterable(iterable, size):
+    iterator = iter(iterable)
+    while chunk := list(islice(iterator, size)):
+        yield chunk
+
+
+def delete_variants(private_config_xml_file, profile, rs_ids_file):
+    with get_mongo_connection_handle(profile, private_config_xml_file) as mongo_conn:
+        db_name = 'eva_accession_sharded'
+
+        chunk_num = 1
+        for chunk in chunked_iterable(rs_ids_file, 1000):
+            logger.info(f"Processing chunk {chunk_num}")
+            chunk_num += 1
+
+            # delete CVE
+            cve_coll = mongo_conn[db_name]['clusteredVariantEntity']
+            delete_result = cve_coll.delete_many({"accession": {"$in": chunk}, "asm": "GCA_000188115.3"})
+            logger.info(f"Deleted {delete_result.deleted_count} documents from clusteredVariantEntity.")
+
+            # remove RS in SVE
+            sve_coll = mongo_conn[db_name]['submittedVariantEntity']
+            update_result = sve_coll.update_many(
+                {"rs": {"$in": chunk}, "seq": "GCA_000188115.3", "study": "PRJEB82793"},
+                {"$unset": {"rs": ""}}
+            )
+            logger.info(
+                f"Updated {update_result.modified_count} documents in submittedVariantEntity (removed rs field).")
+
+            # remove backpropagated RS in SVE
+            update_result = sve_coll.update_many(
+                {"backPropRS": {"$in": chunk}, "seq": "GCA_000188115.2", "study": "PRJEB82793"},
+                {"$unset": {"backPropRS": ""}}
+            )
+            logger.info(
+                f"Updated {update_result.modified_count} documents in submittedVariantEntity (removed backPropRS field).")
+
```

Comment:
I don't think this is the case but in theory these `rs` could have been used by other studies as well. 
Can we check if they are being used anywhere else in these two assemblies ?  
Maybe you can just add a `find` here and If we detect another submitted variant, we should run the same script for these other studies.

Reply:
That's fine then.